tools/exporter/exporter.py

```diff
@@ -566,13 +598,38 @@ def sync_exports(logger, sc_client, settings):
                             media_file = sc_client.get_media(audit_id, media_id)
                             media_export_filename = media_id
                             save_exported_media_to_file(logger, media_export_path, media_file, media_export_filename, extension)
-                        continue
-                    save_exported_document(logger, export_path, export_doc, export_filename, export_format)
+                    elif export_format == 'web-report-link':
+                        web_report_link = sc_client.get_web_report(audit_id)
+                        web_report_data = [template_id,
+                               get_json_property(audit_json, 'template_data', 'metadata', 'name'),
+                               audit_id,
+                               get_json_property(audit_json, 'audit_data', 'name'),
+                               web_report_link]
+                        save_web_report_link_to_file(logger, export_path, web_report_data)
                 logger.debug('setting last modified to ' + audit['modified_at'])
                 update_sync_marker_file(audit['modified_at'])
             else:
                 logger.info('Audit\'s modified_at value is less than {0} seconds in the past, skipping for now!'.format(media_sync_offset))
 
+
+def get_json_property(obj, *args):
```

Comment:
There is in the csvExporter.py. It is a static method in the `CsvExporter` class. To resolve this, I pulled `get_json_property` out of `CsvExporter` and made it a standalone function, so that it is available to the code in `exporter.py`